covid_19/covid19_and_era5_matching.ipynb

```diff
@@ -0,0 +1,7632 @@
+{
```

Comment:
<p>very cool! It will be exciting to revisit this analysis with higher frequency climate data but that can wait.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NCAR/Pangeo_KONQUER/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>